### ISSUE TITLE
Migrate Old Attribution Data Into Segment

### ISSUE THREAD

**Issue by NiaStorj** [storj/storj#4904]
We want to send archival data of old accounts to Segment so that Hubspot can be populated with historic data on which customers are associated to Partners

Blocked by: https://github.com/storj/storj/issues/4903.  Needs to be completed first

Business Objective/Issue:

Acceptance Criteria:

**Comment by NiaStorj** [comment]
ticket no longer needed.
